script/Arcueid/Day 5/05_00_ARC05_1B.txt

```diff
@@ -346,17 +347,17 @@ Then while I was still in a state of shock, you put an end to my vital organs wi
 [sha:01c7d5d0d6769fc11f45f891eff0bbfcc56a8b1b]{
 -- Page 61, Offset 8742.
 -- どのくらい完璧だったかっていうと、あの時の志貴をそのままキャンバスに閉じ込めれば、それ以上はないほどの<芸術品|え>になるぐらい、完璧だった。
-If you were to ask me how perfect it was, I'd say it was so perfect that if you had captured that moment on canvas, it would have become a work of art unmatched by any other.
+If you were to ask me how perfect it was, I'd say it was so perfect that if you had captured that moment on a canvas, it would've become a work of art unmatched by any other.
 }
 [sha:0ee5b0c41b486e1cc0d9c1d2862f62df254a9ea8]{
 -- Page 61, Offset 8743.
 -- わたしがまだ正常なままだったら、お城に持ち帰って飾ってあげたいぐらい」
-If I had still been in one piece, I would have taken you back to the castle as a decoration."
+If I were still in my usual state, I would've taken you back to my castle as a souvenir."
```

Comment:
Kinda like the still in one piece liberty unless we think it's not the actual intent here

Reply:
my read on this was that she was talking about how shiki killing her made her act not 正常  in the behavioral sense, though i might be reading into it too much

Reply:
Hmm, maybe, it just seems weird to talk about her frame of mind at the time when in fact her mind at the time was in several pieces on the floor.

Reply:
idm reverting it if we're sure, cause like i said i might just be misinterpreting it

Reply:
I can't be 100% sure, paging Dr. @Petrikow

Reply:
I'm mostly certain the reasoning she gives for not being able to take him back to her castle is because... well... summoning the Millennium Castle is something she can only do at her full power.

Reply:
so more like full power/peak performance then